guides/common/modules/proc_registering-project-as-a-keycloak-client.adoc

```diff
@@ -1,16 +1,16 @@
 [id="registering-{project-context}-as-a-keycloak-client_{context}"]
-= Registering {Project} as a {Keycloak} client
-Use this procedure to register {Project} to {Keycloak} as a client and configure {Project} to use {Keycloak} as an authentication source.
+= Registering {Project} as a {keycloak-wildfly} client
```

Comment:
would be great if the file name matched the title more closely to differentiate between wildfly and quarkus when editing files.

Reply:
This is actually yet another module used in both wildfly and keycloak scenarios. So my bad, this should say {keycloak} rather than {keycloak-wildfly}. Updated here and in a few other places.